crates/namada/src/ledger/native_vp/masp.rs

```diff
@@ -270,9 +270,12 @@ where
                     .ctx
                     .read_post::<IndexedTx>(pin_keys.first().unwrap())?
                 {
-                    Some(IndexedTx { height, index })
-                        if height == self.ctx.get_block_height()?
-                            && index == self.ctx.get_tx_index()? => {}
+                    Some(IndexedTx {
+                        height,
+                        index,
+                        is_wrapper: false,
```

Comment:
Should we maybe avoid to write this field in storage? I think we always write false anyway for both fee unshielding and proper masp inner txs